packages/components/src/mobile/bottom-sheet/index.native.js

```diff
@@ -435,7 +444,7 @@ class BottomSheet extends Component {
 
 		let listStyle = {};
 		if ( isFullScreen ) {
-			listStyle = { flexGrow: 1 };
+			listStyle = { flexGrow: 1, maxHeight };
```

Comment:
we need to add a `maxHeight` to the styling to make sure the bottom sheet content is scrollable.